filing_history_descriptions.yml

```diff
@@ -35,6 +40,9 @@ description:
     'liquidation-compulsory-winding-up-order-with-case-start-date' : "**Order of court** to wind up on {case_start_date}"
     'liquidation-compulsory-winding-up-order' : "**Order of court** to wind up"
     'liquidation-in-administration-appointment-of-administrator' : "**Appointment of an administrator**"
+    'liquidation-in-administration-appointment-of-a-replacement-or-additional-administrator' : "**Notice of appointment** of a replacement or additional administrator"
+    'liquidation-in-administration-order-limiting-disclosure-of-statement-of-affairs-or-proposals' : "**Notice of order limiting disclosure** of statement of affairs or proposals"
+    'liquidation-in-administration-recission-or-amendment-of-order-limiting-disclosure-of-statement-of-affairs-or-proposals' : "**Notice of rescission or amendment of order limiting disclosure** of statement of affairs or proposals in administration"
```

Comment:
It was agreed not to edit this because the changes had already been published on the forum.